lms/static/js/spec/dashboard/track_events_spec.js

```diff
@@ -14,70 +14,51 @@
-                //Verify that analytics events fire when the 'Learned about verified track' link is clicked.
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.verified-info'),
-                    'edx.bi.dashboard.verified_info_link.clicked',
-                    window.edx.dashboard.generateTrackProperties
-                );
-            });
+                var verifiedInfoLinklArgs = [$('.verified-info'),
+                                             'edx.bi.dashboard.verified_info_link.clicked',
+                                             window.edx.dashboard.generateTrackProperties];
 
-            it('sends an analytics event when the user clicks find courses button', function() {
-                // Verify that analytics events fire when the 'user clicks find the course' button.
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.btn-find-courses'),
-                    'edx.bi.dashboard.find_courses_button.clicked',
-                    {
-                        category: 'dashboard',
-                        label: null
-                    }
-                );
-            });
+                var findCourseButtonArgs = [$('.btn-find-courses'),
+                                            'edx.bi.dashboard.find_courses_button.clicked',
+                                            {
+                                                category: 'dashboard',
+                                                label: null
+                                            }];
 
-            it('sends an analytics event when the user clicks the \'View XSeries Details\' button', function() {
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.xseries-action .btn'),
-                    'edx.bi.dashboard.xseries_cta_message.clicked',
-                    window.edx.dashboard.generateProgramProperties
-                );
+                var xseriersActionButtonArgs = [$('.xseries-action .btn'),
+                                                'edx.bi.dashboard.xseries_cta_message.clicked',
+                                                window.edx.dashboard.generateProgramProperties]
+
+                var expectedArgsForTrackLinkCall = [courseTitleLinkArgs,
+                                                    courseImageLinkArgs,
+                                                    enterCourseLinkArgs,
+                                                    courseOptionDropdownClickedArgs,
+                                                    verifiedInfoLinklArgs,
+                                                    findCourseButtonArgs,
+                                                    xseriersActionButtonArgs];
+
+                for (var index=0; index< expectedArgsForTrackLinkCall.length; index ++){
```

Comment:
i'm wondering if this line will result in a jshint violation. if it does, please fix it.